src/gui/widgets.py

```diff
@@ -312,10 +312,11 @@ def copy(selected: List[QModelIndex]):
         self.__do_with_selected_comment_row(copy)
 
     def add_comment(self, comment_type: str, comment_text: str = "", time: str = None,
-                    sort: bool = True, will_change_qc=True, edit_mode_active=True) -> None:
+                    sort: bool = True, will_change_qc=True, edit_mode_active=True, resize_columns=True) -> None:
```

Comment:
Resize_columns can be omitted at all since it is a leftover from before the column was able to resize itself properly.

Instead a resize trigger is necessary after the preference window is closed, since the comment types may change.